src/dataIntegration/transcriptomics/mCADRE/mCADRE.m

```diff
@@ -0,0 +1,126 @@
+function [tissueModel, coreRxn, nonCoreRxn,	zeroExpRxns, pruneTime, cRes] = mCADRE(model, expressionRxns, threshold_high, protectedRxns, checkFunctionality, eta, tol, ubiquityScore, confidenceScores)
+% Use the mCADRE algorithm (`Wang et al., 2012`) to extract a context

```

Comment:
OK, admittedly, this is odd for mCADRE, not sure, whether preprocessing should be outsourced...